application-analytics-default/src/test/java/com/xwiki/analytics/internal/MostViewedJsonNormaliserTest.java

```diff
@@ -62,21 +63,33 @@ public class MostViewedJsonNormaliserTest
     @Test
     public void testObjectJSONResponse() throws Exception
```

Comment:
You don't have to specify the `test` part, is redundant since you have the annotation.
What it is done is to name a test after the method you are testing, like `methodName` + `particularities`. Here maybe you could have `normalizeDataWithJSONResponse`. The same name should be used for the JSON file used, to do the matching easier when something is checked.

The same applies to the tests below.